iModelCore/libsrc/libxml2/vendor/libxml.h

```diff
@@ -34,22 +34,43 @@
 #else
     /* BENTLEY CHANGES */
     #if defined (_MSC_VER)
-        #include "include/win32config.h"
+        #include "win32/win32config.h"
     #else
         #include "../BeLibxml-unix-config.h"
+        #define XML_SYSCONFDIR "/etc"
```

Comment:
this was added by myself to get this to work in linux/mac. I don't think its the correct placement, @tcobbs-bentley I know you resolved this issue on your side. Where did you put it/any suggestions

Reply:
i also don't like introducing changes to src code, but I just realized we already have our own set of custom changes to this file, so maybe this is fine? Or maybe worth seeing why/if these changes are still needed?

Reply:
(With the ifndef wrapper.)

Reply:
oh sorry duh. Didn't realize that was our, but windows config came with the pkg